server/src/apps/queue/queue.ts

```diff
@@ -0,0 +1,62 @@
+import { Console } from 'console';
+import { Student } from '../student/student'
+
+class Queue {
```

Comment:
Just to confirm, we're going to just be passing this as an object to the associated Booth class for the company right?